client/gutenberg/extensions/contact-form/editor.js

```diff
@@ -1,365 +1,10 @@
 /** @format */
 
-/**
- * External dependencies
- */
-import { getBlockType, createBlock } from '@wordpress/blocks';
-import { Fragment } from '@wordpress/element';
-import { InnerBlocks } from '@wordpress/editor';
-
 /**
  * Internal dependencies
  */
-import './editor.scss';
-import JetpackContactForm from './components/jetpack-contact-form';
-import JetpackField from './components/jetpack-field';
-import JetpackFieldTextarea from './components/jetpack-field-textarea';
-import JetpackFieldCheckbox from './components/jetpack-field-checkbox';
-import JetpackFieldMultiple from './components/jetpack-field-multiple';
-import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
-import renderMaterialIcon from 'gutenberg/extensions/presets/jetpack/utils/render-material-icon';
+import * as form from './index';
```

Comment:
I'd prefer named imports here (`import { fields, name, settings } from './index'`) since it makes clearer what it is that we're requiring from that file.